test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -37,6 +37,7 @@ public override void Setup()
             StartDynamo(dynamoSettings);
         }
 
+        [TearDown]
```

Comment:
did you find you need this for some reason? I think it's marked as teardown in the base class - but nunit / inheritance behavior always requires debugging for me to understand.

Reply:
I guess it doesn't matter. I think you're right; if it's overriding the base class method and it has the attribute in the base class, it isn't needed. No harm in having it here as well though.

Reply:
I think I'll remove it on second thoughts. One less file change in this PR :)